### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jeremykubica**
- Add release notes for v1.0.0
- Change all references of kbmod and KBMoD to KBMOD for consistency
- Change to readthedocs theme (as the pydata-sphinx-theme is broken)

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
+  :alt: KBMOD logo
 
 KBMOD (Kernel Based Moving Object Detection) is a GPU-accelerated framework for the detection of slowly moving asteroids within sequences of images. KBMOD enables users to detect moving objects that are too dim to be detected in a single image without requiring source detection in any individual image, nor at any fixed image cadence. KBMOD achieves this by “shift-and-stacking” images for a range of asteroid velocities and orbits without requiring the input images to be transformed to correct for the asteroid motion.
 
-.. Important:: If you use KBMoD for work presented in a publication or talk please help the project via proper citation or acknowledgement **what is proper citation or acknowledgement**
+.. Important:: If you use KBMOD for work presented in a publication or talk please help the project via proper citation or acknowledgement **what is proper citation or acknowledgement** In addition, if you use the known object matching functionality please cite either `IMCCE's SkyBoT VO tool <https://vo.imcce.fr/webservices/skybot/>`_ (Berthier et. al. 2006) and JPL’s SSD (Solar System Dynamics) `API service <https://ssd.jpl.nasa.gov/>`_ as appropriate.
```
Comment (delucchi-cmu) [inline docs/source/index.rst:12]: Is the `**what is proper citation or acknowledgement**` supposed to be a link or header or TODO?

Comment (delucchi-cmu) [inline docs/source/index.rst:12]:
```suggestion
.. Important:: If you use KBMOD for work presented in a publication or talk please help the project via proper citation or acknowledgement **what is proper citation or acknowledgement** In addition, if you use the known object matching functionality please cite either `IMCCE's SkyBoT VO tool <https://vo.imcce.fr/webservices/skybot/>`_ (Berthier et. al. 2006) or JPL’s SSD (Solar System Dynamics) `API service <https://ssd.jpl.nasa.gov/>`_ as appropriate.
```

File: setup.cfg [hunk 2]
```diff
@@ -57,5 +57,6 @@ docs =
     sphinx
     numpydoc
     pydata-sphinx-theme
```
Comment (delucchi-cmu) [inline setup.cfg:59]: If pydata-sphinx-theme is broken, should the reference be removed?

**Review by delucchi-cmu — COMMENTED** [review]
Does this address the failing docs build? https://github.com/dirac-institute/kbmod/actions/workflows/build_docs.yaml

File: docs/source/index.rst (hunk 1 above)
Comment (jeremykubica) [reply docs/source/index.rst:12]: I have a thread out with the KBMOD team to figure that out and will update this as soon as I have a respond. For now it is a TODO.

File: setup.cfg (hunk 2 above)
Comment (jeremykubica) [reply setup.cfg:59]: Good call. Done

**Comment by jeremykubica** [comment]
With respect to the question of fixing the broken build: Not entirely. This fixes the error in the docs config (the module not found), but doesn't fix what looks like a permissions error on epyc.

**Review by DinoBektesevic — APPROVED** [review]
I'll approve this because fixing documentation the right way greatly exceeds the scope. 

Two things that I will note:
  * changing themes doesn't always work flawlessly, a lot of markdown navigation can display very differently and make it confusing to navigate the docs. Now we have "Overview" with "Input Files" and then "Input Files" themselves alongside every module API documented at the top level of the documentation. 
  * This is partly my fault - the original Sphinx PR https://github.com/dirac-institute/kbmod/pull/184 was supposed to be just an example, not a complete canonical documentation layout. We should keep that in mind and rework the docs
 
The pydata theme problem was fixed in https://github.com/pydata/pydata-sphinx-theme/issues/1094. Nonetheless we should add this tidbit to fix it and replace that with our logo.
```
html_theme_options = {
   "logo": {
      "image_light": "logo-light.png",
      "image_dark": "logo-dark.png",
   }
}
```
(this is more of a note to myself)

So approved, but I'm opening an issue to flesh out the documentation templates and layouts.

**Pull request merged**